crates/bevy_core_pipeline/src/smaa/smaa.wgsl

```diff
@@ -64,7 +64,7 @@
  *                           |output|
  *
  * Note that each [pass] has its own vertex and pixel shader. Remember to use
- * oversized triangles instead of quads to avoid overshading along the
+ * oversized triangles instead of quads to avoid extra shading along the
```

Comment:
overdraw. this is technically not precisely overdraw but its close enough. also it should say `fullscreen triangle` not `oversized triangles` (not plural)